### PR TITLE
Create localisation UI

### PR DESCRIPTION
**Pull request by andy-t-wang**
Will create one more PR for creating a new draft. This will involve creating a hasura custom action

https://github.com/user-attachments/assets/4d9f78c9-5dbd-4778-b07b-a4cb0a3e7685

### CODE REVIEW

**Review by igorosip0v — APPROVED** [review]

File: hasura/metadata/databases/default/tables/public_localisations.yaml [hunk 1]
```diff
@@ -1,3 +1,44 @@
 table:
   name: localisations
   schema: public
+insert_permissions:
+  - role: user
+    permission:
+      check: {}
```
[the remark this answers is not in the document]
Comment (andy-t-wang) [reply hasura/metadata/databases/default/tables/public_localisations.yaml:7]: ah yeah good point

[the remark this answers is not in the document]
Comment (andy-t-wang) [reply hasura/metadata/databases/default/tables/public_localisations.yaml:7]: I will include in my update pr

[the remark this answers is not in the document]
Comment (andy-t-wang) [reply hasura/metadata/databases/default/tables/public_localisations.yaml:7]: added in a new pr

[the remark this answers is not in the document]
Comment (andy-t-wang) [reply hasura/metadata/databases/default/tables/public_localisations.yaml]: Added in a pr, tagged you and merged

**Pull request merged**